src/python/pants/option/global_options.py

```diff
@@ -253,12 +253,21 @@ def register_bootstrap_options(cls, register):
                   'Later files override earlier ones.')
     register('--pythonpath', advanced=True, type=list,
              help='Add these directories to PYTHONPATH to search for plugins.')
-    register('--target-spec-file', type=list, dest='positional_arg_files', daemon=False,
+    register('--target-spec-file', type=list, dest='spec_files', daemon=False,
              removal_version='1.25.0.dev2',
```

Comment:
So, while we don't need to "wait until this deprecation" to start the next one, it's possible that people have already made this edit, and would now need to make another. Consider bumping this either to match, or at least by one version?